Integrations/Gmail/Gmail.py

```diff
@@ -259,12 +361,13 @@ def emails_to_entry(title, raw_emails, format_data, mailbox):
         'Contents': emails,
         'ReadableContentsFormat': formats['markdown'],
         'HumanReadable': tableToMarkdown(title, emails, headers[format_data]),
-        'EntryContext': {'Gmail(val.ID && val.ID == obj.ID)': emails}
+        'EntryContext': {'Gmail(val.ID && val.ID == obj.ID)': emails,
+                         'Email(val.Date == obj.Date && val.Subject == obj.Subject)': context_email}
```

Comment:
`context_email` contain only the last email (in case of a list, you ignore all except the last one).

Reply:
Why not add to Email Context the ID field to make sure we rely on unique values as is done with Gmail Context?